app/controllers/profile_controller.rb

```diff
@@ -3,7 +3,12 @@ class ProfileController < ApplicationController
   before_action :require_user
 
   def index
-    @user = current_user
+    # binding.pry
+    if params[:id]
+      @user = User.find(params[:id])
+    else
+      @user = current_user
+    end
```

Comment:
This was kind of weird. I had to set up a different route for the admin to view user order profile pages. Since the profile page is set by the session I had to create a way to specify that it was not the current_user profile page. This tests that if the profile page index is sent a param then the @user is set by the params[:id] but if no :id is sent in the params then the profile is the current_user.

Reply:
Nice job figuring this one out. 🥇

Reply:
Nice work!